Libplanet/Blockchain/BlockChain.Swap.cs

```diff
@@ -359,5 +370,12 @@ private static Block<T> FindTopCommon(BlockChain<T> c1, BlockChain<T> c2)
 
             return null;
         }
+
+        private static IAction ToAction(IValue plainValue)
+        {
+            var action = new T();
+            action.LoadPlainValue(plainValue);
+            return action;
+        }
```

Comment:
I kinda see the necessity but this should get removed at some point down the line. 🙄

Reply:
Yes, you're right. I just wanted to separate works into a PR to introduce `IActionEvaluation` and another one to update renderers' code.